rosbridge_client/scripts/rosbridge_websocket.py

```diff
@@ -230,7 +232,9 @@ def handle_compression_offers(offers):
             if isinstance(offer, PerMessageDeflateOffer):
                 return PerMessageDeflateOfferAccept(offer)
 
-    factory = RosbridgeWebSocketClientFactory(socketServerURL)
+    headers = {'Authorization: Bearer ': jwt_token}
```

Comment:
This almost right but the : here you need to concat Bearer and jwt_token as the value